packages/forklift-console-plugin/src/modules/Overview/dynamic-plugin.ts

```diff
@@ -37,4 +38,14 @@ export const extensions: EncodedExtension[] = [
       required: ['CAN_LIST_NS'],
     },
   } as EncodedExtension<HrefNavItem>,
+
+  {
+    type: 'console.context-provider',
+    properties: {
+      provider: { $codeRef: 'OverviewStateContextProvider.CreateOverviewStateContextProvider' },
```

Comment:
```suggestion
      provider: { $codeRef: 'OverviewStateContextProvider.CreateOverviewContextProvider' },
```

i prefer a shorter name here,  the 'state' does not give new information, I know it's stores a state because it's a context